catalog/imaging/pom.xml

```diff
@@ -56,6 +56,8 @@
         <module>imaging-transformer-chipping</module>
         <module>imaging-app</module>
```

Comment:
I am running into a unit test failure because the imaging-nitf-api and imaging-nitf-imp modules do not come before imaging-app, in fact, I believe imaging-app should be the last module built and all other modules should build before it.